server/index.js

```diff
@@ -57,7 +57,9 @@ function questionAuthoringAuth(req, res, next) {
 // api routes
 // helper for db connection pooling
 function queryDatabase(text, values, cb) {
-  pg.connect(process.env.DATABASE_URL +'?ssl=true', function(err, client, done) {
+  var addSSL = process.env.NODE_ENV === 'development' ? '' : '?ssl=true';
+  pg.connect(process.env.DATABASE_URL + addSSL, function(err, client, done) {
+    console.log(err)
```

Comment:
I'd either remove this, or wrap it in an `if` so it only logs if there actually is an error.